src/CMakeLists.txt

```diff
@@ -81,3 +82,63 @@ if (RUBY_FOUND)
    	  --gtest_output=xml:${CMAKE_BINARY_DIR}/test_results/${test}rb.xml)
   endforeach()
 endif()
+
+#################################
+# Create and install Python interfaces
+# Example usage
+# $ export PYTHONPATH=/ws/install/lib/python/:$PYTHONPATH
+if (PYTHONLIBS_FOUND)
+  set_source_files_properties(python/python.i PROPERTIES CPLUSPLUS ON)
+  set_source_files_properties(python/python.i PROPERTIES SWIG_FLAGS "-includeall")
+  set_source_files_properties(python/python.i PROPERTIES SWIG_MODULE_NAME "math")
+  set(SWIG_PY_LIB pymath)
+  set(SWIG_PY_LIB_OUTPUT math)
+
+  set(CMAKE_SWIG_OUTDIR "${CMAKE_INSTALL_PREFIX}/lib/python/ignition")
```

Comment:
I believe this line is causing the abi-checker build to fail, as it is trying to write files during build directly to the install folder. I think somewhere in the build folder would be more suitable?

~~~
Error creating directory "/usr/local/source_branch/lib/python/ignition".
src/CMakeFiles/_pymath.dir/build.make:62: recipe for target '/usr/local/source_branch/lib/python/ignition/pythonPYTHON_wrap.cxx' failed
make[2]: *** [/usr/local/source_branch/lib/python/ignition/pythonPYTHON_wrap.cxx] Error 1
CMakeFiles/Makefile2:326: recipe for target 'src/CMakeFiles/_pymath.dir/all' failed
make[1]: *** [src/CMakeFiles/_pymath.dir/all] Error 2
~~~

[![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=ignition_math-abichecker-any_to_any-ubuntu_auto-amd64&build=421)](https://build.osrfoundation.org/job/ignition_math-abichecker-any_to_any-ubuntu_auto-amd64/421/) https://build.osrfoundation.org/job/ignition_math-abichecker-any_to_any-ubuntu_auto-amd64/421/

Reply:
regular Ubuntu build is failing too

Reply:
Fixed at e26eeb4